nomenclature/cli.py

```diff
@@ -15,7 +15,9 @@ def cli_valid_yaml(path: Path):
 
 @cli.command("validate-project")
 @click.argument("path", type=click.Path(exists=True, path_type=Path))
-def cli_valid_project(path: Path):
+@click.option("--mappings", default="mappings")
```

Comment:
Although technically redundant it's always nice to tell the users the exact data type you expect. 
```suggestion
@click.option("--mappings", type=str, default="mappings")
```